packages/salesforcedx-vscode-apex/src/index.ts

```diff
@@ -104,6 +105,14 @@ export async function activate(context: vscode.ExtensionContext) {
 function registerCommands(
   extensionContext: vscode.ExtensionContext
 ): vscode.Disposable {
+  // Colorize code coverage
+  const statusBarToggle = new StatusBarToggle();
+  const colorizer = new CodeCoverage(statusBarToggle);
+  const colorizerCmd = vscode.commands.registerCommand(
```

Comment:
As per the pattern i am seeing for other commands, i think this could be renamed to forceApexToggleColorizerCmd.